index.html

```diff
@@ -286,6 +286,42 @@ <h2 class="topic">
             </ul>
         </div>
     </section>
+
+    <footer id="contact" class="footer">
+        <div class="contact-me">
+            <h2>Contact me</h2>
+            <p>
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d love
+                to help with it
+            </p>
```

Comment:
- [ ] Please 🙏🏽 correct the margin of the text to be similar to the Figma template.

![image](https://user-images.githubusercontent.com/58642949/186796662-ff7226b3-3c3e-468c-8b62-e3070a64d51d.png)

Reply:
good job following the previous reviewer's suggestions